src/Tests/DcsBios/DCSBIOSControlLocatorTests.cs

```diff
@@ -0,0 +1,363 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using ClassLibraryCommon;
+using DCS_BIOS.ControlLocator;
+using DCS_BIOS.Json;
+using DCS_BIOS.Serialized;
+using Xunit;
+
+namespace DCSFPTests.DcsBios
+{
+    public class DCSBIOSControlLocatorTests
+    {
+        //private readonly int _a10C = 5; // A-10C Thunderbolt/II
+        //private readonly string _dcsbiosPath = Environment.ExpandEnvironmentVariables(@"%USERPROFILE%\Saved Games\DCS\Scripts\DCS-BIOS\doc\json");
+        //private readonly string _arc21025KhzSelectorId = "ARC210_25KHZ_SEL";
+
+        [Theory]
+        [InlineData(@"%USERPROFILE%\Saved Games\DCS\Scripts\DCS-BIOS\doc\json", 5, "ARC210_25KHZ_SEL")] // A-10C Thunderbolt/II
```

Comment:
I see the point but I'm not sure if it's a good idead to reference files directly on the file system that are not included in the solution. Suppose you want to launch the tests on a remote machine like a compilation server or if the user has not the file in the directory, then the test can't run.
For testing purpose, I would include those files in the solution somewhere, preferably in a kind of `Resources' folder.

Reply:
The code is written so that it won't run if there isn't a dcs-bios installation at that location. I just verified that it works. To include the files in the solution will defeat the purpose IMO as changes to dcs-bios won't be captured unless you remember to *duplicate* the files.